packages/mobile/src/backup/utils.ts

```diff
@@ -162,3 +170,14 @@ export function isValidBackupPhrase(phrase: string) {
 export function isValidSocialBackupPhrase(phrase: string) {
   return isValidMnemonic(phrase, 13)
 }
+
+export async function encryptMnemonic(phrase: string) {
+  const pepper = await retrieveOrGeneratePepper()
+  return CryptoJS.AES.encrypt(phrase, pepper).toString()
```

Comment:
Shouldn't we encrypt the mnemonic with both the pepper and passwordHash?

Reply:
I'll be changing this to use the password instead, which is pin + pepepr